assets/js/blocks/cart/inner-blocks/proceed-to-checkout-block/block.tsx

```diff
@@ -16,15 +16,22 @@ import { CHECKOUT_STORE_KEY } from '@woocommerce/block-data';
  */
 import './style.scss';
 
+export const defaultButtonLabel = __(
+	'Proceed to Checkout',
+	'woo-gutenberg-products-block'
+);
+
```

Comment:
@alexflorisca I wonder if we should move the button label into a designated file, e.g. `constants.tsx`. We're doing the same for the product elements currently:

https://github.com/woocommerce/woocommerce-blocks/blob/8c857e2422d68479e7f5d798184fa19931e1eb07/assets/js/atomic/blocks/product-elements/image/constants.tsx#L7-L17

That way, we keep `block.tsx` lean, and we can still use the translatable string both in `block.tsx` and in `attributes.tsx`.

Reply:
We only move things to constants file when they're dynamic, this string should be in attributes.jsx or block.json